src/leveled_ebloom.erl

```diff
@@ -302,13 +302,7 @@ split_builder_speed_tester() ->
     Timings =
         lists:map(
             fun(HashList) ->
-                SlotCount =
-                    case length(HashList) of
-                        0 ->
-                            0;
-                        L ->
-                            min(128, max(2, (L - 1) div 512))
-                    end,
+                SlotCount = min(128, max(2, (length(HashList) - 1) div 512)),
```

Comment:
This gives `SlotCount` 2 for empty `HashList` which it before also did for `HashList` of length 1, so probably this is just fine.

Reply:
It was a test to help with assessing the performance of changes to the code, and the condition of a zero length hashlist never occurred in the test (it was copied from the create_bloom function)